crates/teloxide-core/src/types/bot_command.rs

```diff
@@ -11,6 +11,11 @@ pub struct BotCommand {
     /// Can contain only lowercase English letters, digits and underscores.
     pub command: String,
 
+    /// Aliases of the command, each 1-32 characters.
+    ///
+    /// Can contain only lowercase English letters, digits and underscores.
```

Comment:
Why?

Reply:
In the telegram docs he said that [here](https://core.telegram.org/bots/features#commands). Should I remove `English letters`?

Reply:
Also, it's considered a command in the end, so I think they have the same description, right?

Reply:
Opps :)